functionnal_tests/cypress/integration/workspace/create_workspace_spec.js

```diff
@@ -25,19 +25,19 @@ describe('Workspace', () => {
       .should('exist')
   })
 
-  describe('Dashboard', () => {
-    describe('Creating two new workspaces', () => {
-      const createOneWorkspace = (cy, workspaceName) => {
-        return cy.get('[data-cy="sidebarCreateWorkspaceBtn"]').should('be.visible').click()
-          .get('[data-cy="createcontent__form__input"]').should('be.visible').type(workspaceName)
-          .get('[data-cy="popup__createcontent__form__button"]').should('be.visible').click()
-      }
+  const createOneWorkspace = (cy, workspaceName) => {
+    return cy.get('[data-cy="sidebarCreateWorkspaceBtn"]').click()
+      .get('[data-cy="createcontent__form__input"]').type(workspaceName)
+      .get('[data-cy="popup__createcontent__form__button"]').click()
+  }
 
-      const getWorkspaceItemByName = (workspaceTitle) => (
-        cy.get(`.sidebar__content__navigation__workspace__item__name[title="${workspaceTitle}"]`)
-          .parents('li.sidebar__content__navigation__workspace__item')
-      )
+  const getWorkspaceItemByName = (workspaceTitle) => (
```

Comment:
give cy as a parameter as you did for createOneWorkspace.
It's better to have functions that don't use resources from outside of themselves